components/scream/src/share/io/scream_scorpio_interface.hpp

```diff
@@ -2,43 +2,61 @@
 #define SCREAM_SCORPIO_INTERFACE_HPP
 
 #include "share/scream_types.hpp"
+#include <vector>
+
+/* C++/F90 bridge to F90 SCORPIO routines */
 
 // TODO, figure out a better way to define netCDF output type for fields
 #ifdef SCREAM_CONFIG_IS_CMAKE
 #  ifdef SCREAM_DOUBLE_PRECISION
-  const int PIO_REAL = 6;
+  static constexpr int PIO_REAL = 6;
 #  else
-  const int PIO_REAL = 5;
+  static constexpr int PIO_REAL = 5;
 #  endif // SCREAM_DOUBLE_PRECISION
 #else // SCREAM_CONFIG_IS_CMAKE
-  const int PIO_REAL = 6;
+  static constexpr int PIO_REAL = 6;
 #endif // SCREAM_CONFIG_IS_CMAKE
-const int PIO_INT = 4;
+static constexpr int PIO_INT = 4;
 
-using scream::Real;
-using scream::Int;
 namespace scream {
 namespace scorpio {
-
-  void eam_init_pio_subsystem(const int mpicom, const int compid, const bool local);
+  /* All scorpio usage requires that the pio_subsystem is initialized. Happens only once per simulation */
```

Comment:
Not important at all, but you should feel free to use C++-style comments (//) instead of C ones (/* */), even in C code.

Reply:
Good point.  I think this file will be significantly revised when we do away from the C++ -> F90 bridge.  That would be a good time to also update the comments and probably change their format.